PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.core/src/com/microsoft/azuretools/core/utils/AzureAbstractHandler.java

```diff
@@ -39,12 +39,19 @@ public Object execute(ExecutionEvent event) throws ExecutionException {
 
 		final Map<String, String> properties = new HashMap<>();
 		try {
-			properties.put("CategoryId", event.getCommand().getCategory().getId());
-			properties.put("Category", event.getCommand().getCategory().getName());
-			properties.put("CommandId", event.getCommand().getId());
-			properties.put("Text", event.getCommand().getName());
+			Command cmd = event.getCommand();
```

Comment:
Suggest adding `final` keyword as possible as you can to prevent unintended changes. It's one of the best practices in java.